ecl/eclcc/eclcc.cpp

```diff
@@ -639,7 +644,12 @@ int main(int argc, const char *argv[])
         //In release mode exit without calling all the clean up code.
         //It is faster, and it helps avoid potential crashes if there are active objects which depend on objects in file hook dlls.
         fflush(NULL);
+
```

Comment:
A better fix may be to include this whole if() (lines 637..643) in a #ifndef _COVERAGE.